tests/modules/services/window-managers/sway/sway-invalid-config.nix

```diff
@@ -0,0 +1,12 @@
+{ config, lib, ... }:
+
+lib.mkIf config.test.enableBig {
+  wayland.windowManager.sway = {
+    enable = true;
+    checkConfig = true;
+  };
+
+  nmt.script = ''
+    assertFileExists home-files/.config/sway/config
```

Comment:
Why this test is called "invalid-config" but expect a success? Is it possible to add some fail tests to ensure the check actually do something?